src/screens/Lending/Dashboard/Rates.js

```diff
@@ -1,20 +1,20 @@
 // @flow
-import React, { useMemo, useCallback, useState } from "react";
-import { View, StyleSheet, FlatList } from "react-native";
-import { BigNumber } from "bignumber.js";
-import { Trans } from "react-i18next";
-import _ from "lodash";
-import type { AccountLikeArray } from "@ledgerhq/live-common/lib/types";
 import type { CurrentRate } from "@ledgerhq/live-common/lib/families/ethereum/modules/compound";
+import type { AccountLikeArray } from "@ledgerhq/live-common/lib/types";
 // import { formatShort } from "@ledgerhq/live-common/lib/currencies";
 import { useNavigation, useTheme } from "@react-navigation/native";
-import LText from "../../../components/LText";
-import CurrencyUnitValue from "../../../components/CurrencyUnitValue";
+import { BigNumber } from "bignumber.js";
+import _ from "lodash";
+import React, { useCallback, useMemo, useState } from "react";
```

Comment:
How come so many new imports in this file? They don't appear to be getting used @Justkant